content/docs/dashboard/features.md

```diff
@@ -16,32 +16,68 @@ The user interface is still under development and therefore does not offer so ma
 
+
+## Creating an application
+
+In Oakestra there is the principle of applications and services, within one application there can be several services and one user can create several applications.
+
+In order to create a service, an application must first be created.  This is very simple, you only need to specify the name, the namespace and an optional description.
+
+![](/add-app.gif)
+
 ## Creating a service
 
-Creating a service is the main task of the dashboard, as described in other parts of the documentation, 
-an SLA definition must be created for it. This can be easily done using the various input fields 
-in the dashboard. 
+Creating a service is the main task of the dashboard, as described in other parts of the documentation,
+an SLA definition must be created for it. This can be easily done using the various input fields
+in the dashboard.
 
-To do this, first, create an application, then you can create a service in this application by 
-entering the various values in the form. The dashboard then creates the SLA deployment descriptor 
+To do this, first create an application or select the appropriate application in which a new service should be created.  Then you can create a service in this application by entering the various values in the form. The dashboard then creates the SLA provisioning descriptor
 based on your input and sends it to the root orchestrator.
 
+![](/create-service.gif)
 
-## Organizations
+If you already have other SLA configurations, you can upload this 
+configuration to the dashboard, and it will send everything to the root.
 
-Organizations ensure smooth collaboration within a team in Oakestra. 
-Members of an organization have access to all applications created within that organization, 
-and they can use the resources provided within the organization to deploy new services.
+Please note that the JSON file should have the following format.
 
-### Root Organization
 
-When the root orchestrator is started for the first time, a root organization 
-is also created with the admin, and each user is automatically part of this root organization. 
-However, users can also be part of other organizations.
-The root organization has specific properties. 
-Unlike other organizations, where a user can view all applications within the organization 
-and share resources, in the root organization, 
-only the applications that a user has created can be viewed.
+```
+{
+  "microservices": [
+    {
+        "microserviceID": "",
+        "microservice_name": "Demo",
+        "microservice_namespace": "demo",
+        "virtualization": "container",
+        "cmd": [],
+        "memory": 100,
+        "vcpus": 1,
+        "vgpus": 0,
+        "vtpus": 0,
+        "bandwidth_in": 0,
+        "bandwidth_out": 0,
+        "storage": 0,
+        "code": "docker.io/library/nginx:latest",
+        "state": "",
+        "port": "8080",
+        "added_files": []
+        ....
+    }
+  ]
+}
+
```

Comment:
The deployment descriptor is different from the example descriptor on the wiki already (https://www.oakestra.io/docs/getstarted/get-started-app/). Can we keep things consistent?